darwin/dataset/download_manager.py

```diff
@@ -95,11 +95,13 @@ def download_all_images_from_annotations(
 
         if not force_replace:
             # Check the planned path for the image against the existing images
-            planned_image_path = (
-                images_path
-                / Path(annotation.remote_path.lstrip("/\\")).resolve().absolute()
-                / Path(annotation.filename)
-            )
+            filename = annotation.filename
```

Comment:
Is there a reason it was `Path(annotation.filename)` before?

Reply:
Good spot - I don't believe so. `annotation.filename` gives type `str` but it can still be used to construct `Path` objects in this way

However, there is no harm in making it type `Path`, so I have kept this behaviour